### PR TITLE
Allow to 'yield' events

### PR DESCRIPTION
**Pull request by wiibaa**
PR of https://github.com/logstash-plugins/logstash-filter-ruby/issues/6#issuecomment-97320945

### CODE REVIEW

File: lib/logstash/filters/ruby.rb [hunk 1]
```diff
@@ -13,6 +13,15 @@
 #       } 
 #     } 
 #
+# If you need to create additional events, it cannot be done as in other filters where you would use `yield`,
+# you must use a specific syntax `new_event_block.call(event)` like in this example duplicating the input event
```
Comment (jordansissel) [inline lib/logstash/filters/ruby.rb]: What about having a helper method `yield_event` that folks can call? `yield_event(whatever)` thoughts?

**Comment by jordansissel** [comment]
Idea LGTM. Code LGTM. Haven't tested. I'll test once we figure out the naming/interface details (yield_whatever, etc)

**Comment by talevy** [comment]
can we rebase this to keep this moving forward?

**Comment by wiibaa** [comment]
@talevy rebase done.

**Comment by talevy** [comment]
LGTM

**Comment by jordansissel** [comment]
I'm don't likethe name `new_event_block` because it's a bit long, but as this filter is intended only for hacks, I'm ok with it.

**Comment by jordansissel** [comment]
lgtm

**Comment by talevy** [comment]
I kind of like the rather explicit name for the sake of uniqueness. I also agree, if someone really wants this feature, I'm sure the name is not an issue to them

**Pull request merged**